server/emulator.go

```diff
@@ -84,49 +89,119 @@ func (m EmulatorAPIServer) CommitBlock(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+}
+func (m EmulatorAPIServer) SnapshotList(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	badgerStore := (*m.storage).Store().(*badger.Store)
```

Comment:
will this work with multiple store implementations? should you do a check whether the type is indeed badger.

Reply:
Actually my plan is to migrate snapshots to sqlite, they are small, and self contained. 

Badger has windows big file size problem (or windows has the problem according to badger guys)

Some portable snapshot format (still we can use git, but also can provide a download api ) can be pretty useful.